src/pages/CompanyCourseManagement/CCMPage.jsx

```diff
@@ -0,0 +1,97 @@
+// Import necessary components from Material-UI library
+import Button from "@mui/material/Button";
+import { List, ListItem, ListItemText } from "@mui/material";
+// Import custom PageLayout component
+import PageLayout from "../../components/PageLayout";
+
+// CCMPage component definition
+function CCMPage() {
+  // Return JSX markup for the CCMPage component
+  return (
+    // Use the custom PageLayout component with specific title and viewport settings
+    <PageLayout title="CCM" viewportPage>
+
+      {/* Container for the page content */}
+      <div
+        sx={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          justifyContent: "center",
+          height: "100vh",
+          padding: "20px", 
+        }}
+      >
+
+        {/* Button for adding training */}
+        <Button
+          variant="contained"
+          color="primary"
+          size="large"
+          sx={{ marginLeft: "16px", marginTop: "16px", alignSelf: "flex-start" }} // Align the button to the top-left
+        >
+          ADD TRAINING
+        </Button>
+
+        {/* Container for the List of trainings */}
+        <div
+          sx={{
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "center",
+            alignItems: "center",
+            width: "100%",
+            flexGrow: 1,
+            padding: "20px", 
+          }}
+        >
+
+          {/* List component to display trainings */}
+          <List
+            sx={{
+              display: "grid",
+              marginTop: "100px",
+              gridTemplateColumns: "repeat(2, 400px)",
+              gap: "80px", // Adjust as needed
+              justifyContent: "center",
+              alignItems: "center",
+              width: "100%",
+            }}
+          >
+
+            {/* Map through an array of training items to create ListItems */}
+            {[1, 2, 3, 4, 5, 6].map((training) => (
```

Comment:
Possibly outsource your [1, 2, 3, 4, 5, 6] to an object that could then also hold the "training" thingy as a value.
Here's a cute smiley for your efforts:
(๑>◡<๑)

Reply:
Is this related to the props feature? If so, please do check the changes made to the code. I'm still not sure if it's done correctly